java/code/src/com/suse/manager/webui/services/test/SaltServerActionServiceTest.java

```diff
@@ -200,7 +201,58 @@ public void testPackageUpdate() throws Exception {
         Action updateAction = ActionFactory.lookupById(action.getId());
 
         Map<LocalCall<?>, List<MinionSummary>> result = saltServerActionService.callsForAction(updateAction, minionSummaries);
-        RhnBaseTestCase.assertNotEmpty(result.values());
+        assertEquals(1, result.values().size());
```

Comment:
You assert that it will be 1 value, but you iterate 2 values on next line? :thinking:

Reply:
No, it's a nested structure, so it's taking "1st element of the collection inside the collection", not 2nd element of the 1st collection :)

Reply:
ah lol, right.